docs/en/tutorials/config.md

```diff
@@ -245,180 +166,283 @@ model = dict(
                 type='nms',  # Type of NMS
                 iou_thr=0.5),  # NMS threshold
             max_per_img=100,  # Max number of detections of each image
-            mask_thr_binary=0.5))  # Threshold of mask prediction
+            mask_thr_binary=0.5)))  # Threshold of mask prediction
+```
+
+### Dataset and evaluator config
+
+[Dataloaders](https://pytorch.org/docs/stable/data.html?highlight=data%20loader#torch.utils.data.DataLoader) are required for the training, validation, and testing of the [runner](https://mmengine.readthedocs.io/en/latest/tutorials/runner.html). Dataset and data pipeline need to be set to build the dataloader. Due to the complexity of this part, we use intermediate variables to simplify the writing of dataloader configs.
+
+```python
 dataset_type = 'CocoDataset'  # Dataset type, this will be used to define the dataset
 data_root = 'data/coco/'  # Root path of data
-img_norm_cfg = dict(  # Image normalization config to normalize the input images
-    mean=[123.675, 116.28, 103.53],  # Mean values used to pre-training the pre-trained backbone models
-    std=[58.395, 57.12, 57.375],  # Standard variance used to pre-training the pre-trained backbone models
-    to_rgb=True
-)  # The channel orders of image used to pre-training the pre-trained backbone models
-train_pipeline = [  # Training pipeline
-    dict(type='LoadImageFromFile'),  # First pipeline to load images from file path
+file_client_args = dict(backend='disk')  # file client arguments
```

Comment:
`file client docs` for implementation details.

Reply:
MMEngine currently does not have file client docs. I'll update it later.